cassovary-core/src/main/scala/com/twitter/cassovary/graph/SemiSynchronizedDynamicGraph.scala

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2014 Twitter, Inc.
```

Comment:
Good point.  I meant to change the copyright.  I'm not sure if it should be copyright with our names, or Teapot, or something like Apache Foundation.  Let's talk offline about how to handle copyright.